exporters/otlp/otlplogs/otlplogshttpjson/client.go

```diff
@@ -341,3 +341,7 @@ func (d *client) MarshalLog() interface{} {
 		Insecure: d.cfg.Insecure,
 	}
 }
+
+func init() {
```

Comment:
Init function in all-in-one module will be executed once, only one of the modules will be registered this way.
this solution will not work